Dockerfile

```diff
@@ -7,14 +7,20 @@ ARG BATS_VERSION=1.6.0
 ENV POLICY_PATH="/project"
 
 RUN curl -L https://github.com/open-policy-agent/conftest/releases/download/v"${conftest_version}"/conftest_"${conftest_version}"_Linux_x86_64.tar.gz | tar -xz --no-same-owner -C /usr/bin/ && \
+    curl https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz --output oc.tar.gz && tar -xzvf oc.tar.gz -C /usr/bin && \
```

Comment:
Just realized that the https://mirror.openshift.com/pub/openshift-v4/clients/oc is a [deprecated location](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/oc/%24__DEPRECATED_LOCATION__PLEASE_READ__.txt). I'd suggest using https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz